fl4health/losses/deep_mmd_loss.py

```diff
@@ -47,7 +47,7 @@ def __init__(
         hidden_size: int = 10,
         output_size: int = 50,
         lr: float = 0.001,
-        training: bool = True,
+        training: bool = False,
```

Comment:
With this change, you'll want to update the init documentation default to False.

A more general question is what we expect to happen if the user sets `training = True` in creating the loss? I haven't thought this fully through, but it seems like the user would expect that to mean that the trainable parameters were, in fact, trainable. 

If we want the loss to always start with training off and that we (or the user) will be responsible for turning it on (and later off) at the right times, then we can probably just drop this argument entirely and set `self.training = False` explicitly to avoid any mismatch.

Reply:
yes. Exactly that's why I want to have this off so people need to turn on training to activate it.

On another note training basically activates training of kernel multiple epochs, and then compute loss, however if it is off, it only computes loss.

I agree that we can omit it from input arguments.